tests/integration/targets/test_manage_machine_image/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# defaults file for tests/intergration/test_manage_image
+_resource_prefix: "cloud-gpc-ops"
```

Comment:
using `cloud/gcp`, ansible-test will generate a `resource_prefix` variable
for now as this is comment, you should rather using random value to avoid collision when running 2 iterations of the current integration targets in the same GCP project

Reply:
Done, thanks!